modules/cram/templates/concat_vcf.sh

```diff
@@ -19,7 +19,7 @@ concat () {
 }
 
 bcftools_sort () {
-  ${CMD_BCFTOOLS} sort --output-type z "unsorted_!{vcfOut}" --output "!{vcfOut}"
+  ${CMD_BCFTOOLS} norm --rm-dup all "unsorted_!{vcfOut}" | ${CMD_BCFTOOLS} sort --output-type z --output "!{vcfOut}"
```

Comment:
```suggestion
  ${CMD_BCFTOOLS} norm --do-not-normalize --rm-dup all --no-version --threads "!{task.cpus}" "unsorted_!{vcfOut}" | ${CMD_BCFTOOLS} sort --temp-dir . --max-mem "!{task.memory.toGiga() - 1}" --output-type z --output "!{vcfOut}"
```

- `--do-not-normalize`: because this might interfere with DeepVariant input style expected by GlNexus
- `--no-version`: consistency with other processes
- `--threads`: performance
- `--temp-dir . --max-mem "!{task.memory.toGiga() - 1}"`: make more robust similar to straglr process